File: WpCoreSolution/Presentation/Wp.Web.Api/Models/Admin/Expenses/ExpenseModel.cs
using System;

namespace Wp.Web.Api.Models.Admin
{
    public class ExpenseModel 
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public decimal Amount { get; set; }
        public string Date { get; set; }
        public string Notifications { get; set; }
        public bool IsDebit { get; set; }
        public string Code { get; set; }
        public string TransactionType { get; set; }
        public string ExpenseTags { get; set; }
        public ExpenseCategoryModel ExpenseCategory { get; set; }
        public ExpenseAccountModel ExpenseAccount { get; set; }      
    }   
}

File: wp-be/src/app/admin/expense/expense.model.ts
export class Expense {
    id: number;
    name: string;
    description: string;
    value: number;
    date: string;
    expenseAccount: ExpenseAccount;
    expenseCategory: ExpenseCategory;
}

export class ExpenseAccount {
    name: string;
    description: string;
}

export class ExpenseCategory{
    name: string;
    description: string;
    color: string;
}